### ISSUE TITLE
Change password

### ISSUE THREAD

**Issue by daniyaalk** [drtweety/Calibr#24]
Allow users who've created accounts to change their passwords.

**Comment by daniyaalk** [comment]
Mostly completed, awaiting resolution of #25 for final few lines of code.

Only `TODO` part in `scripts/updatepassword.js` in cbb8c21 is remaining for resolution of this issue
```javascript
$.ajax({
    method: 'POST',
    url: 'app/updatepassword.php',
    data: {
      'password': current_password,
      'new': new_password,
      'repeat': repeat_password
    },
    success: function(data){
      if(data.changed){
        //TODO: add code
      }else{
        console.log(data.messages);
        $.each(data.messages,function(key, value){
          $("#"+value.field+"-password-group").addClass("has-error");
          $("#"+value.field+"-password-group > .help-block").append(value.message+"<br />");
        });
      }
    }
  });
```

I'll finish it once code is added for better alerts system.

**Comment by daniyaalk** [comment]
Resolved in 2eb5f2d
